main.go

```diff
@@ -60,6 +61,7 @@ type Portal struct {
 type SheetGroup struct {
 	AutomatedSheetName string // sheet managed by application
 	SheetName          string
+	EnvSheetNames      []string
```

Comment:
these names are kind of confusing to me. `Env` seems superfluous since everything in this group is environment-specific. So really this amounts to two fields called `SheetName` and `SheetNames`

Reply:
`SheetName` should be named `PortalSheetName`: the name of the sheet that has all accounts in the portal.
`EnvSheetNames` are all the sheets that use a given portal. The testers created 1 sheet per environment for running tests in that environment. For example, 4 environments use `portaldev.cms.gov`. `Env` in `EnvSheetNames` refers to the testing environment, not the portal env.

Reply:
Renamed `SheetName` to `PortalSheetName` and `EnvSheetNames` to `TestingSheetNames`